tensorflow/compiler/mlir/hlo/lib/utils/disc_op_list.cc

```diff
@@ -0,0 +1,17 @@
+#ifdef GET_ELEMENTWISE_OP_LIST
+#undef GET_ELEMENTWISE_OP_LIST
+
```

Comment:
Please document

Reply:
The file name was misleading.
I rename the file as elementwise_op_list.cc. I hope it should be clear enough as the name suggests.